cmake_modules/dependencies/yakkety.cmake

```diff
@@ -0,0 +1 @@
+set ( CPACK_DEBIAN_PACKAGE_DEPENDS  "hpccsystems-platform, ganglia-monitor, ganglia-monitor-python, python-lxml, rrdtool")
```

Comment:
http://packages.ubuntu.com/search?keywords=ganglia-monitor
ganglia-monitor is the actual ganglia package for ubuntu.  hpccsystems-ganglia-monitoring is the name of our package.